src/main/resources/killbill-java/api.mustache

```diff
@@ -2,21 +2,23 @@
 
 package {{package}};
 
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Objects;
 
 {{#imports}}import {{import}};
 {{/imports}}
 
-import com.google.common.collect.Multimap;
-import com.google.common.base.Preconditions;
-import com.google.common.base.MoreObjects;
-import com.google.common.collect.LinkedListMultimap;
-
 import org.killbill.billing.client.Converter;
 import org.killbill.billing.client.KillBillClientException;
 import org.killbill.billing.client.KillBillHttpClient;
 import org.killbill.billing.client.RequestOptions;
 import org.killbill.billing.client.RequestOptions.RequestOptionsBuilder;
 
+import org.killbill.commons.utils.Preconditions;
```

Comment:
Wondering if we should add a hard dependency on this...

Ideally, since the library is embedded in the users application, if would be nice to not force any dependency.

Reply:
`checkNotNull()` added in `api.mustache`